mgmtsystem/scvmm.py

```diff
@@ -148,6 +148,12 @@ def list_vm(self, **kwargs):
             "Select name | ConvertTo-Xml -as String")
         return etree.parse(StringIO(data)).getroot().xpath("./Object/Property[@Name='Name']/text()")
 
+    def list_hosts(self, **kwargs):
+        data = self.run_script(
+            "Get-SCVMHost -VMMServer $scvmm_server |"
+            "Select Name | Where-Object {$_.Name -like 'qeblade*'}| ConvertTo-Xml -as String")
```

Comment:
@psav ,yes it's a bit clumsy, but that helps updating only actual hosts as we also have 'dhcp129*' host on original SCVMM which doesn't work at the moment and both qeblade26 and qeblade33 are used for vm provisioning. and updating Hosts pretty often won't be an issue - that is the only way to know actual free space on the datastore. Before we faced a lot of provisioning errors because of lack or discrepancy of free space on the selected datastore.